build.gradle

```diff
@@ -106,6 +105,8 @@ minecraft {
 //}
 
 dependencies {
+    implementation 'org.jetbrains:annotations:19.0.0'
```

Comment:
What does this provide? I can add to main branch if its useful

Reply:
TBH I'm not sure how that got there.